pkg/controller/servicebindingrequest/mapper.go

```diff
@@ -132,6 +133,8 @@ func (m *sbrRequestMapper) Map(obj handler.MapObject) []reconcile.Request {
 		}
 		log.Debug("current resource is a SBR", "Requests", requests)
 		return requests
+	} else {
```

Comment:
else is redundant here

Reply:
You're right in principle, but in order to make it easier to follow the debug logs more information should be added.